src/components/MARCTable/MARCTableRow.js

```diff
@@ -286,8 +280,8 @@ export const MARCTableRow = ({
                 <FormattedMessage id="ui-data-import.settings.mappingProfile.marcTable.data.find" />
               </Label>
               <TextArea
-                value={dataValue?.find || ''}
-                onChange={e => onDataChange(e, 'find')}
+                value={field.data?.find || ''}
+                onChange={onDataFieldChange('find')}
```

Comment:
I think in the long run string literals params to this function are good to have in constants.

Reply:
I hope when these fields will be wrapped in the Field and connected to the form, `onChange` methods will became unnecessary and literals as well. But if it is not so, then I'll move it to constants

Reply:
Sounds good.